src/config/default-app-config.ts

```diff
@@ -256,7 +256,18 @@ export class DefaultAppConfig implements AppConfig {
       metadataDetailsList: [
         { label: 'Document type', name: 'dc.type' }
       ]
-    }
+    },
+    dynamicFields: [
+      {
+        triggeringField: 'dc.contributor.author',
+        endpoint: 'toto.dspace.com',
```

Comment:
in order to add our new configuration for dynamic fields, I had to modify the "AppConfig" interface. Since "DefaultAppConfig" uses this interface, we need to add a default for our "dynamicFields".

Reply:
UPDATE: I will change it to an empty list, so:
`dynamicFields: []`